.drone.yml

```diff
@@ -0,0 +1,90 @@
+pipeline:
```

Comment:
This is a Drone 0.8 pipeline, not a Drone 1.0  one.

Reply:
Converted to 1.0